src/feb.c

```diff
@@ -1464,6 +1468,31 @@ int API_FUNC qthread_readFE(aligned_t *restrict       dest,
     return QTHREAD_SUCCESS;
 }                      /*}}} */
 
+/* the way this works is that:
+ * 1 - src's FEB state is ignored
+ * 2 - data is copied from src to destination
+ */
+
+int API_FUNC qthread_readXX(aligned_t *restrict       dest,
+                            const aligned_t *restrict src)
+{                      /*{{{ */
+    qthread_t          *me      = qthread_internal_self();
+
+    assert(qthread_library_initialized);
+
+    if (!me) {
+        return qthread_feb_blocker_func(dest, (void *)src, READXX);
```

Comment:
Certainly, you don't need `QTHREAD_FEB_UNIQUERECORD` - that's only there to profile the performance of the central striped hash table (when `FEB_PROFILING` is turned on) - if you don't use that hash table, you shouldn't use `QTHREAD_FEB_UNIQUERECORD` (in my opinion).

As for qthread_debug, in my personal opinion, debugging output should *never* require heavyweight operations - especially not ones that get triggered regardless of whether debugging is turned on or not. I prefer my code lean and mean when debugging isn't on.